resolwe/flow/tests/processes/spawned.yml

```diff
@@ -46,5 +46,5 @@
     program: |
       # TODO: use re-import when added to resolwe-runtime-utils
       # re-import {{ input_file.file_temp }} {{ input_file.file }} "bar" "bar" 0.9
-      mv {{ input_file.file_temp }} {{ input_file.file }}
-      re-save-file saved_file {{ input_file.file }}
+      mv "{{ input_file.file_temp }}" "{{ input_file.file }}"
```

Comment:
What does this change have to do with Django version bump? This is probably something that should be moved into a separate commit?